tests/unit_tests/test_gameconnection.py

```diff
@@ -124,7 +124,7 @@ def test_abort(game_connection, game, players):
 
 async def test_handle_action_GameOption_change_name(game, game_connection):
     await game_connection.handle_action('GameOption', ['Title', 'All welcome'])
-    assert game.name == 'All welcome'
+    assert game.name == game.sanitize('All welcome')
```

Comment:
Calling functions for asserts is bad practice. That adds unnecessary dependencies into the test.

Reply:
I'm encountering a problem with that one where "game.name" becomes a mock() object, because when instantiating a game, the name is automatically sanitized (but here the game is mocked). So I must either run the test like this (so it gives out "mock == mock"), or remove it.
(or else i have no idea)

Reply:
Good lord. That's a horrible test setup, but I guess we got to live with it for now.